src/groundlight/client.py

```diff
@@ -148,18 +157,21 @@ def get_or_create_detector(
             )
         return existing_detector
 
+    @RequestsRetryDecorator()
     def get_image_query(self, id: str) -> ImageQuery:  # pylint: disable=redefined-builtin
         obj = self.image_queries_api.get_image_query(id=id)
         iq = ImageQuery.parse_obj(obj.to_dict())
         return self._fixup_image_query(iq)
 
+    @RequestsRetryDecorator()
     def list_image_queries(self, page: int = 1, page_size: int = 10) -> PaginatedImageQueryList:
         obj = self.image_queries_api.list_image_queries(page=page, page_size=page_size)
         image_queries = PaginatedImageQueryList.parse_obj(obj.to_dict())
         if image_queries.results is not None:
             image_queries.results = [self._fixup_image_query(iq) for iq in image_queries.results]
         return image_queries
 
+    @RequestsRetryDecorator()
```

Comment:
This is too high a level for the retry decorator, because there's a bunch of logic going on inside here.  This method first calls the `POST` API method to create the imagequery, and then polls for a confident response.  If any of those return a 500, then the whole thing will retry, which is not what we want.  

Imagine if we get a 500 on one of the polling requests which is checking to see if a human label has showed up yet.  Having a retry at this level would result in a second imagequery being created.